packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```
@@ -35,19 +35,32 @@
     }
 
     &__field {
+        position: absolute;
+        inset: 0;
         min-width: 0; /* this is required to reset input's default width */
+        padding-left: 4rem;
         display: flex;
         flex-grow: 1;
         font-family: inherit;
         outline: 0;
         font-size: 1.4rem;
+        background-color: transparent;
         color: var(--system-light-2-general-text, #333);
         transition: border-color 0.2s;
-        cursor: pointer;
-    }
+        cursor: unset;
+        user-select: none;
+
+        &::selection {
+            background-color: transparent;
+        }
```

You can actually combine these two classnames. And a question, is the mozilla vendor prefix required?

```suggestion
        &::selection, &::-moz-selection {
            background-color: transparent;
        }
```

---

These cannot be combined unfortunately ([source](https://stackoverflow.com/questions/16982449/why-isnt-it-possible-to-combine-vendor-specific-pseudo-elements-classes-into-on))